programs/cleos/main.cpp

```diff
@@ -1416,9 +1416,9 @@ int main( int argc, char** argv ) {
       return 1;
    } catch (connection_exception& e) {
       auto errorString = e.to_detail_string();
-         if (errorString.find(fc::json::to_string(port)) != string::npos) {
+         if (errorString.find(host + ":" + fc::json::to_string(port)) != string::npos) {
             std::cerr << localized("Failed to connect to nodeos at ${ip}:${port}; is nodeos running?", ("ip", host)("port", port)) << std::endl;
-         } else if (errorString.find(fc::json::to_string(wallet_port)) != string::npos) {
+         } else if (errorString.find(wallet_host + ":" + fc::json::to_string(wallet_port)) != string::npos) {
```

Comment:
Should that be "wallet-host" instead of "wallet_host"?

Reply:
These lines don't even exist after the merged HTTPS support, I had to refactor this logic in to what I hope is a little cleaner.

I'm not sure the original defect still exists in my refactor or not.